src/databricks/sql/client.py

```diff
@@ -297,6 +298,53 @@ def _check_not_closed(self):
         if not self.open:
             raise Error("Attempting operation on closed cursor")
 
+    def _handle_staging_operation(self):
+        """Make HTTP request using instructions provided by server"""
+
+        row = self.active_result_set.fetchone()
+
+        # TODO: Handle headers. What format will gateway send? json? plaintext?
+        operation, presigned_url, local_file, headers = (
+            row.operation,
+            row.presignedUrl,
+            row.localFile,
+            None,
+        )
+
+        operation_map = {
```

Comment:
This will be expanded to include REMOVE and GET

Reply:
as this is constant I wonder if we should define this outside of the method? or is the python culture do this inside method?

Reply:
REMOVE is supported, just making sure it will be a separate PR or we will add it here ?

Reply:
Yeah sorry I know it's confusing. The way the tickets for this were first made it seemed like we wanted separate PR's for:

- PUT
- GET
- REMOVE
- Restrict PUT to `baseUploadsPath`

But I think it should really just be one PR which keeps it from being so confusing / wasting reviewer time.

Reply:
In 55525cbf8660c7d47a628a2c072610b45c212ee8 I refactored to stub in the GET and REMOVE behaviours. All these will be included in this PR.